techniques/html/H48.html

```diff
@@ -155,11 +159,11 @@ <h3>Example 4: Contact information using a description list</h3>
   &lt;/div&gt;
 &lt;/dl&gt;</code></pre>
 
-<p class="working-example">This is shown in the <a href="../../working-examples/description-list-contact-info/">working example of contact information using a description list</a>.</p>
+<p class="working-example">これは、<a href="https://www.w3.org/WAI/WCAG22/working-examples/description-list-contact-info/">working example of contact information using a description list</a> に示されている。</p>
       </section>
       <section class="example" id="example-5-using-lists-to-group-links">
-         <h3>Example 5: Using lists to group links</h3>
-         <p>In this example the links are grouped using the <code class="language-html">ul</code> and <code class="language-html">li</code> elements.</p>
+         <h3>事例 5: リストを用いてリンクをグループ化する</h3>
+         <p>この事例では、リンクが <code class="language-html">ul</code> 要素と <code class="language-html">li</code> 要素を用いてグループ化されている。/p>
```

Comment:
`</p>` の閉じタグが欠損していました。
```suggestion
         <p>この事例では、リンクが <code class="language-html">ul</code> 要素と <code class="language-html">li</code> 要素を用いてグループ化されている。</p>
```